frontend/src/metabase/admin/settings/components/widgets/PublicLinksListing/PublicLinksListing.styled.ts

```diff
@@ -0,0 +1,12 @@
+import styled from "@emotion/styled";
+import Button from "metabase/core/components/Button";
+import { color } from "metabase/lib/colors";
+
+export const IconButton = styled(Button)`
```

Comment:
There is `IconButtonWrapper` that is supposed to do that. Extending the `Button` and overwriting its styles is discouraged